pymongo/event_listeners.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2020-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from pymongo import monitoring
+from pymongo.monitoring import ConnectionPoolListener
+
+
+class CommandListener(monitoring.CommandListener):
```

Comment:
What do you think about naming these classes "Logger" instead of "Listener" (`CommandLogger`, `HeartbeatLogger`, etc..) so that users are not confused about the duplicate class names?

Reply:
Done.